data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1704 @@
+	on complete
+		payment 132000
+		conversation
+			`More blue-uniformed workers are waiting when you land, though these uniforms consist of incredibly thick, bulky coats. They take the crates out of your ships while the doctors are directed to a hospital building, and <payment> are transferred to you when the workers are done. They thank you for your help and ask that you look for them in Kimek worlds if you wish to continue helping.`
+
+mission "Resistance: Charity 3"
+	name "Charity Delivery"
+	description "Bring <cargo> to <destination>"
+	cargo "charity supplies" 46
+	source
+		near "14 Pole" 3 5
+	destination "Remote Blue"
+	to offer
+		random < 70
+		has "Resistance: Charity 2: done"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+	on offer
+		conversation
+			`A group of Kimek in the blue uniforms approach you in the spaceport. "Greetings, Captain <last>," one of them says. "Helped our charity before, you have. Interested in more work, are you?"`
+			choice
+				`	"What do you need me to do?"`
+				`	"Sorry, I'm not interested in helping anymore."`
+					decline
+			`	"Lack many of their preferred diet options, the Arachi and Saryds on <planet> do," they explain. "Enough to order it from off-world, the poorer of them do not have, so sending them a care package, our group is."`
+			`	They show you to a pile of containers and crates near the hangars, and you have them load them into your ship. "Once on <planet>, you are, pay you <payment>, we will. Thank you for the help, Captain." They wish you a safe trip, and vanish into the spaceport halls.`
+				accept
+	on complete
+		payment 261000
+
+mission "Resistance: Grenades"
+	name "Grenade Smuggling"
+	description "Bring 100 fragmentation grenades from human space to <destination>."
+	landing
+	source "Remote Blue"
+	destination "Mebla's Portion"
+	to offer
+		has "Resistance: Charity 3: done"
+		not "Heliarch License: done"
+	on offer
+		conversation
+			`You drop off the crates full of charity goods, and the spaceport workers start getting them up on vehicles to transport them to the poor. One of the workers transfers 261,000 credits to your account, thanking you for the help.`
+			branch metchiree
+				has "Resistance: Questions: done"
+			`	As they move the last few crates, you hear a rapid tapping approaching from behind, and when you turn to investigate a Kimek tackles you. Some more join in, covering your head with some dark sack, and start pushing you around once they restrain you. You call out for help, but nobody comes. The Kimek keep pushing and tugging at you as they bring you around the city. After many minutes, you hear some doors close behind you, and your captors force you into some cramped room or compartment, where you can't stand up straight due to the short ceiling. You feel as if the room was tugged down in a moment, and when it stops, you're walked around for a while longer, before the Kimek seat you and remove the sack from your head.`
```

Comment:
They are long, but I don't think it's necessary to break these two up with a choice. From what I recall they weren't going out of my screen? Even if they are, I'd rather shorten the two paragraphs if it's really necessary, rather than giving the player a choice during a time where they're supposed to be helpless.

Reply:
Makes sense.